modules/ximgproc/include/opencv2/ximgproc/ridgefilter.hpp

```diff
@@ -0,0 +1,102 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Use short license headers instead of outdated:
```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```